tailcfg/tailcfg.go

```diff
@@ -1984,6 +1985,16 @@ const (
 	// rather than one big /10 CGNAT route. At most one of this or
 	// NodeAttrOneCGNATEnable may be set; if neither are, it's automatic.
 	NodeAttrOneCGNATDisable = "one-cgnat?v=false"
+
+	// NodeAttrPeerMTUEnable makes the client do path MTU discovery to its
```

Comment:
these aren't imperative commands, only add a single attr `NodeAttrPeerMTUEnable = "peer-mtu"` and use its presence to check if it should be enabled/disabled

Reply:
Here are the stages:

1. PeerMTU disabled on client by default, but turned on by control per tailnet policy.
2. PeerMTU enabled on client by default, but turned off by per-tailnet policy. <- enable flag removed here
3. If there is a bug, PeerMTU disabled by control regardless of per-tailnet policy.

So I think I want two nodeAttrs until step 2, at which point we can remove PeerMTUEnable and have PeerMTUDisable only for catastrophic bugs.

Reply:
add the first nodeAttr to enable in step 1
add the second nodeAttr to disable, remove first nodeAttr in step 2
As each step would require a capVer bump, adding those values then makes sense.

If we add two now, ordering of the nodeAttrs delivered to the node ends up mattering which is not desired.

Reply:
Okay, that makes sense! Will do when I next have the chance.